rasa/shared/utils/io.py

```diff
@@ -328,9 +328,9 @@ def read_yaml(content: Text, reader_type: Union[Text, List[Text]] = "safe") -> A
     Raises:
         ruamel.yaml.parser.ParserError: If there was an error when parsing the YAML.
     """
-    fix_yaml_loader()
+    # fix_yaml_loader()
 
-    replace_environment_variables()
+    # replace_environment_variables()
```

Comment:
this brought down the reading times with woz from 1h to 3.6 minutes. In my opinion we only need to replace env variables for config files (`endpoints.yml`, `config.yml`, `credentials.yml`, but not for stories / nlu)